misc-utils/findmnt.c

```diff
@@ -1492,10 +1492,11 @@ int main(int argc, char *argv[])
 			outarg = optarg;
 			break;
 		case FINDMNT_OPT_OUTPUT_ALL:
-			for (ncolumns = 0; ncolumns < ARRAY_SIZE(infos); ncolumns++) {
-				if (is_tabdiff_column(ncolumns))
+			ncolumns = 0;
+			for (int i = 0; i < ARRAY_SIZE(infos); i++) {
```

Comment:
Note that we do not use `for(int i ...)` C++ism in the code. and ARRAY_SIZE() is based on size_t, i should be size_t too.